client/src/pages/login.js

```diff
@@ -63,6 +63,7 @@ function Login() {
       .then(res => {
         if (res.status > 299) throw Error(res.message + "");
         window.sessionStorage.setItem("AuthToken", res.token);
+        window.sessionStorage.setItem("userId",JSON.stringify(res.user));
```

Comment:
We should probably save the user as 'user' instead of 'userId', otherwise it may get confusing.